lib/zhr_devs/submissions/submission_identity.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ZhrDevs.Submissions.SubmissionIdentity do
```

Comment:
I understand what this module is meant to do, but I don't understand WHAT IT IS (are modules like these called in a particular way in CRSQ methodology?) and HOW IS IT USED?

Reply:
https://github.com/commanded/commanded/blob/master/guides/Commands.md#custom-aggregate-identity

Reply:
I understand it now. Can you please link this in the moduledoc of this module and the link to explanation of identity macro (in the same document) into router module where we call the macro, with a small explanation like "in this module, to route events to the correct place, we need to identify the consumers, to do that, we use identify macro via module XYZ. See URL".

Sorry for these requests, I'm just 100% sure that this stuff is something other contributors may get confused about.